### ISSUE TITLE
Kendo grid KOtemplate is messed up when we refer the kendo widget specific js as individual reference

### ISSUE THREAD

**Issue by gouthamlp** [kendo-labs/knockout-kendo#181]
Hi there,

I have a page where I am loading kendo autocomplete and kendo grid as separate widget specific js along knockout kendo bindings
and when I do that grid template get messed up(UI is not proper). if I change the order of the reference it works fine 
i.e kendo.autocomplete and the kendo.grid and if kendo.grid referenced first then grid UI is messed up and that this happens only when 
grid data is set using kendoDataSource otherwise its fine.

Brief on why I am loading 2 individual components(build using grunt).
We have app where we are using limited kendo widgets and grid is one among them and we have a infra 
where user can load any kendo widget which is not available in this case autocomplete.
![grid loads fine when order of kendo reference is changed](https://cloud.githubusercontent.com/assets/12123336/7338410/b850d2d4-ec68-11e4-9dbf-c27ccae65d05.PNG)
![grid ui is messed up](https://cloud.githubusercontent.com/assets/12123336/7338411/b856d9cc-ec68-11e4-80f3-a27a4746c963.PNG)

It does not works when I have references order like this
 jquery-1.9.1.min.js
knockout-3.1.0.min.js
kendo.grid.js
kendo.autocomplete.js
knockout-kendo.js
	
but works when order of the reference is changed
jquery-1.9.1.min.js
knockout-3.1.0.min.js
kendo.autocomplete.js
kendo.grid.js
knockout-kendo.js

**Comment by IanYates** [comment]
According to the Kendo docs - http://docs.telerik.com/kendo-ui/install/custom#grid - I believe you need to include the JS files in a specific order to make the grid work properly.  The example on that page has kendo.grid.js at the end, which is what I'd expect since it can rely on the other kendo components as embedded editors and thus they ought to be registered first (so the grid can see what's available and configure itself accordingly).

**Comment by Rakesh-gene** [comment]
As per the documentation of the telerik kendo-ui page, kendo.autocomplete.js should be declared before kendo.grid.js.

**Comment by gouthamlp** [comment]
Hi Guys,

Thanks for the reply. I wanted to make to clear kendo.grid and kendo.autocomplete are not original kendo source.
These are build using git bash(using grunt command)
grunt custom:grid
which gives kendo.custom.js(I renamed it as kendo.grid.js)
and same goes for autocomplete (renamed as kendo.autocomplete.js)

I have created sample kendo page where I am using these reference to load autocomplete and grid widgets without knockout binding(just kendo).
and it works fine irrespective of which order those files are referenced.
this issue happens only when I am using knockout-kendo library.
Today I spent sometime debugging what is causing this issue and here are my findings
in kendo.grid.js there is a function DataSource.create which has a return statement
return dataSource instanceof DataSource ? dataSource : new DataSource(dataSource);
when referenced in this order kendo.autocomplete.js and kendo.grid.js
this check dataSource instanceof DataSource is true
if the order is kendo.grid.js and kendo.autocomplete.js
then this check is dataSource instanceof DataSource is false
but I see both cases dataSource is instance of DataSource object only
also weird thing is if i change the code to something like this
return dataSource instanceof kendo.data.DataSource ? dataSource : new DataSource(dataSource);
it always works irrespective of the order of the kendo files.
I have created seperate sample page for kendo and knockout-kendo 
which would demonstrate the issue. not sure how I can share it here
